11ty/definitions/suicide.md

```diff
@@ -5,4 +5,25 @@ defined: false
 flag:
   level: warning
   text: content warning
+reading:
+  - text: "Mental Health Foundation: Suicide"
+  href: https://www.mentalhealth.org.uk/a-to-z/s/suicide
+  - text: "Samaritans: Best Practice Suicide Reporting Tips"
+  href: https://www.samaritans.org/about-samaritans/media-guidelines/best-practice-suicide-reporting-tips/
 ---
+the deliberate act of voluntarily ending one's own life. Motivation for suicide can be the result of conscious or unconscious thoughts and feelings.
+
+## Issues
+
+Commonly referred to in conversation and journalism as "committed suicide" or similar. This has negative connotations with criminality and judgement, and places blame on the individual. Can also be used in combination with a related measure of "success" which promotes suicide as a desirable result.
+
+## Impact
+
+Unless used with sensitivity and care, discussion of suicide can reinforce negative stereotypes and stigma around mental illness. Sensationalist reporting of suicide also risks inadvertently promoting suicide.
+
+## Usage Tips
+
+- When discussing suicide, avoid mentioning details of methods used and suggesting that these are "easy" or "painless" to reduce the risk of imitation. Also avoid suggesting that suicide is an "easy way out".
+- Avoid speculation over possible triggers: Suicide is complex and not necessarily the result of a single event. Attempting to explain away suicide with a specific cause downplays the wider circumstances and the impact on all those affected.
+- Rather than saying that a person "committed suicide", instead say that a person "died by suicide". This frames suicide as a symptom of wider conditions and the individual as a victim rather than a perpetrator.
```

Comment:
```suggestion
- Instead of saying that a person "committed suicide", say that a person "died by suicide." Framing suicide as a symptom of wider/unknown-to-us conditions validates the individual as a victim, rather than a perpetrator.
```